src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FieldTypes.scala

```diff
@@ -10,12 +11,30 @@ import scala.util.{Failure, Success, Try}
 
 object FieldTypes {
   private case object UuidCoercionViolation extends ValueCoercionViolation("Valid UUID expected")
+  private case object LocalDateTimeCoercionViolation extends ValueCoercionViolation("Valid Local Date Time expected")
 
   private def parseUuid(s: String): Either[ValueCoercionViolation, UUID] = Try(UUID.fromString(s)) match {
     case Success(uuid) => Right(uuid)
     case Failure(_) => Left(UuidCoercionViolation)
   }
 
+  private def parseDate(s: String): Either[ValueCoercionViolation, LocalDateTime] = Try(LocalDateTime.parse(s)) match {
+    case Success(localDateTime) => Right(localDateTime)
+    case Failure(_) => Left(LocalDateTimeCoercionViolation)
+  }
+
+  implicit val LocalDateTimeType: ScalarType[LocalDateTime] = ScalarType[LocalDateTime]("LocalDateTime",
+    coerceOutput = (ldt, _) => ldt.toString,
```

Comment:
It looks like the output of `toString` depends on the precision of the input: https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html#toString--

Is it possible to format it to a specific precision? Are the database timestamps in seconds? And can we include `Z` as the UTC time zone so there's no ambiguity in the API?

Reply:
_Is it possible to format it to a specific precision?_ - Yes. What precision should be used, hours/seconds/milliseconds?
_Are the database timestamps in seconds?_ - DB timestamp precision is to milliseconds
_And can we include `Z` as the UTC time zone so there's no ambiguity in the API?_ - Not sure what you mean here, do you mean to include the 'UTC' timezone as part of the outputted Local Datetime string, ie: "2021-01-27 13:35:15 UTC"